src/linters/eslint.js

```diff
@@ -43,17 +43,31 @@ class ESLint {
 	 * @param {string} prefix - Prefix to the lint command
 	 * @returns {{status: number, stdout: string, stderr: string}} - Output of the lint command
 	 */
-	static lint(dir, extensions, args = "", fix = false, prefix = "") {
-		const extensionsArg = extensions.map((ext) => `.${ext}`).join(",");
-		const fixArg = fix ? "--fix" : "";
+	static lint(dir, extensions = [], args = "", fix = false, prefix = "") {
 		const commandPrefix = prefix || getNpmBinCommand(dir);
-		return run(
-			`${commandPrefix} eslint --ext ${extensionsArg} ${fixArg} --no-color --format json ${args} "."`,
-			{
-				dir,
-				ignoreErrors: true,
-			},
-		);
+		const { stdout: version } = run(`${commandPrefix} eslint -v`, { dir });
+		const isV9 = version.startsWith("v9.");
```

Comment:
This appears to also be required for eslint version 8.x.y as well.

It would be better to parse out the major version `^v(\d+)\.\d+.\d+`, and check it it's `>= 8`.